src/simplicial-hott/05-segal-types.rzk.md

```diff
@@ -375,32 +375,35 @@ all $x$ then $(x : X) → A x$ is a Segal type.
   (A : (_ : X) -> U)
   (fiberwise-is-segal-A : (x : X) -> is-local-horn-inclusion (A x))
   : is-local-horn-inclusion ((x : X) -> A x)
-  := triple-compose-is-equiv
-       (Δ² -> ((x : X) -> A x) )
-       ((x : X) -> Δ² -> A x )
-       ((x : X) -> Λ -> A x )
-       (Λ -> ((x : X) -> A x) )
-        (\ g -> \ x -> \ {t : 2 * 2 | Δ² t} -> g t x) -- first equivalence
-            (second (flip-ext-fun
-              (2 * 2)
-              Δ² (\{t : 2 * 2 | Δ² t} -> BOT)
-              X
-              (\ {t : 2 * 2 | Δ² t} -> A)
-              (\ {t : 2 * 2 | BOT} -> recBOT)))
-        (\ h -> \ x -> \ {t : 2 * 2 | Λ t} -> h x t) -- second equivalence
-          (second (function-equiv-fibered-equiv
-              funext
-              X
-              (\ x -> {t : 2 * 2 | Δ² t} -> A x )
-              (\ x -> {t : 2 * 2 | Λ t} -> A x )
-              (\ x -> (horn-restriction (A x) , fiberwise-is-segal-A x))))
-        (\ h -> \{t : 2 * 2 | Λ t} -> \ x -> (h x) t) -- third equivalence
-          (second(flip-ext-fun-inv
-            (2 * 2)
-            Λ (\ {t : 2 * 2 | Λ t} -> BOT)
-            X
-            (\ {t : 2 * 2 | Λ t} -> A)
-            (\ {t : 2 * 2 | BOT} -> recBOT)))
+  :=
+    triple-compose-is-equiv
+      ( Δ² -> ((x : X) -> A x) )
+      ( (x : X) -> Δ² -> A x )
+      ( (x : X) -> Λ -> A x )
+      ( Λ -> ((x : X) -> A x) )
+      ( \ g x t -> g t x) -- first equivalence
+      ( second (flip-ext-fun
+        ( 2 * 2)
+        ( Δ²)
+        ( \ t -> BOT)
+        ( X)
+        (\ t -> A)
+        (\ t -> recBOT)))
+      ( \ h x t -> h x t) -- second equivalence
+      ( second (equiv-function-equiv-fibered
+        ( funext)
+        ( X)
+        ( \ x -> Δ² -> A x )
+        ( \ x -> Λ -> A x )
```

Comment:
I think here it makes sense to introduce redundant parentheses to (more clearly) separate the type from the arguments of a lambda:

```suggestion
        ( \ x -> (Δ² -> A x) )
        ( \ x -> (Λ -> A x) )
```

@fredrik-bakke what do you think? Should this be added to the style guide?